x/gov/README.md

```diff
@@ -1298,6 +1311,19 @@ Example (`software-upgrade`):
 simd tx gov submit-legacy-proposal software-upgrade v2 --title="Test Proposal" --description="testing, testing, 1, 2, 3" --upgrade-height 1000000 --from cosmos1..
 ```
 
+#### cancel-proposal
+
+The `cancel-proposal` command allows the proposer to cancel the goverance proposal before voting period ends.
```

Comment:
Can you add here too what happens with the deposit? Or somewhere else in the README (as you did in the UPGRADING.md)